Assets/Tests/PlayMode/CombinedManagers/LaborOrderManager_VM_Tests.cs

```diff
@@ -0,0 +1,124 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Reflection;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using UnityEditor;
+
+public class LaborOrderManager_VM_Tests
+{
+    private LaborOrderManager_VM laborOrderManager;
+
+    [SetUp]
+    public void SetUp()
+    {
+        GameObject laborOrderManagerPrefab = Resources.Load<GameObject>("LaborOrderManager_VM");
```

Comment:
Are you sure this prefab exists in the Resources folder?

Reply:
This was a left-over play mode script which I have removed.